src/handlers/TechRecordsListHandler.ts

```diff
@@ -135,21 +142,43 @@ export class TechRecordsListHandler<T extends Vehicle> {
       );
     }
 
-    if (techRecordItem.techRecord.length <= 0) {
-      throw new HTTPError(404, HTTPRESPONSE.RESOURCE_NOT_FOUND);
-    }
-
     return techRecordItem;
   }
 
-  private formatTechRecordItemsForResponse(techRecordItems: T[]) {
-    const recordsToReturn = [];
-    return techRecordItems.map(this.formatTechRecordItemForResponse);
-    // for (let techRecordItem of techRecordItems) {
-    //   techRecordItem = this.formatTechRecordItemForResponse(techRecordItem);
-    //   recordsToReturn.push(techRecordItem);
-    // }
-    // return recordsToReturn;
+  private multipleRecordsWithSameSystemNumber(techRecordItems: T[]): boolean {
+    const uniqueSystemNumbers = new Set(techRecordItems.map((item) => item.systemNumber));
```

Comment:
Is it possible for a search by system number to return more than one system number? Would the set not always have `size === 1`?

Reply:
No as when a vehicle's vin has changed it's represented by multiple records in DynamoDB with the same system number but different vin as vin is the range key.